build.py

```diff
@@ -309,7 +309,7 @@ def RunCMake(context, extraArgs=None, stages=None):
 def RunCTest(context, extraArgs=None):
     buildDir = context.buildDir
     variant = BuildVariant(context)
-    numJobs = context.numJobs
+    numJobs = 1
```

Comment:
No problem figuring this out later, but internally we're able to run our tests in parallel. Can we leave a `#TODO` comment here so that we (hopefully) remember to revisit this?